app/controllers/tweets_controller.rb

```diff
@@ -0,0 +1,7 @@
+class TweetsController < ApplicationController
+  def profile
```

Comment:
Please make your resources as RESTful as possible. Consider naming this action `index` instead of `profile`, because this looks like a list  of tweets to me.